runtime/mainnet/src/weights/xcm/mod.rs

```diff
@@ -0,0 +1,312 @@
+mod pallet_xcm_benchmarks_fungible;
+mod pallet_xcm_benchmarks_generic;
+
+use alloc::vec::Vec;
+
+use frame_support::BoundedVec;
+use pallet_xcm_benchmarks_fungible::WeightInfo as XcmFungibleWeight;
+use pallet_xcm_benchmarks_generic::WeightInfo as XcmGeneric;
+use xcm::{
+	latest::{prelude::*, AssetTransferFilter},
+	DoubleEncoded,
+};
+
+use crate::{config::xcm::MaxAssetsIntoHolding, Runtime};
+
+trait WeighAssets {
+	fn weigh_assets(&self, weight: Weight) -> Weight;
+}
+
+const MAX_ASSETS: u64 = 100;
+
+impl WeighAssets for AssetFilter {
```

Comment:
Implemented this based on: https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/asset-hubs/asset-hub-polkadot/src/weights/xcm/mod.rs

Might need further consideration.

Reply:
can we include that link the in the source code please

Reply:
[fe79c4b](https://github.com/r0gue-io/pop-node/pull/479/commits/fe79c4b7fcabdf6c818b513ac03060da22adf681)